packages/react-error-overlay/src/containers/StackFrame.js

```diff
@@ -168,7 +157,7 @@ class StackFrame extends Component<Props, State> {
       }
     }
 
-    const canOpenInEditor = this.getEndpointUrl() !== null;
+    const canOpenInEditor = this.getErrorLocation() !== null;
```

Comment:
Shouldn't this also still consider if overlay has a callback set? We don't want to style the text as clickable if it's a noop (if my understanding is correct here).

Can we do away with the `openInEditorListener` and `openInEditor` indirection and simply pass what the user has sent all the way through to the component so we can check if it's null?

I'd imagine we could just call `update()` when the callback is set.

Reply:
Yes, I think we should. I'll fix this.